boot/environ/app/bootmgr/bootmgr.c

```diff
@@ -2158,7 +2158,7 @@ BmDisplayDumpError (
     if (BmpInternalBootError)
     {
         /* Return it -- but it's a pointer? */
-        return (ULONG)BmpInternalBootError; // ???
+        return (ULONG_PTR)BmpInternalBootError; // ???
```

Comment:
You'd need to fix the return type too, then. Though this line is probably a bug anyway heh.

Reply:
Yeah, looks like the author of this code was confused about his own code. I'll let him deal with it.